themes/pom.xml

```diff
@@ -11,6 +11,109 @@
     <artifactId>keycloak-themes</artifactId>
     <name>Keycloak Themes</name>
     <description/>
+    <packaging>jar</packaging>
+
+    <properties>
+        <dir.common>src/main/resources/theme/keycloak/common/resources</dir.common>
+        <dir.account2>src/main/resources/theme/keycloak-preview/account/resources</dir.account2>
+        <args.npm.install>ci --no-optional --ignore-scripts</args.npm.install>
+    </properties>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>1.8.0</version>
```

Comment:
Do we need to override the version from the parent pom for this plugin? Can we not just change the version in the parent? Also, is the a feature or bug fix in 1.8.0 that we need? The newest version in brew is currently 1.7.6.

Reply:
Parent POM specifies 1.5, hadn't seen that. I don't know of a specific reason to use later than that version, but since we've been using 1.8.0 [since before my PR](https://github.com/keycloak/keycloak/blob/89fc0e1fcc3df43e4a8e89c817b897f8aa9a5d60/themes/pom.xml#L141), I don't want to change that now. I'll update the parent POM's version, since the plugin isn't used anywhere but themes.

The nice thing about building in PNC is it will import the third party artifacts for us, unlike Brew.